src/eko/msbar_masses.py

```diff
@@ -1,10 +1,20 @@
-r"""
-This module contains the |RGE| for the |MSbar| masses
-"""
+r"""|RGE| for the |MSbar| masses."""
+from math import nan
```

Comment:
Why not `math.nan`? In case of doubt, I prefer the simpler one, and `math` is standard library.

However, NumPy scalars are also problematic for serialization, so the alternative would be `float("nan")`, but it is lengthier to type many times.

Reply:
fine, you just had numpy imported here